src/components/modals/Send/index.js

```diff
@@ -165,7 +167,14 @@ class SendModal extends PureComponent<Props, State<*>> {
 
   handleChangeAppOpened = (isAppOpened: boolean) => this.setState({ isAppOpened })
   handleChangeTransaction = transaction => this.setState({ transaction })
-  handleRetry = () => this.setState({ error: null, errorSteps: [] })
+  handleRetry = () => {
+    this.setState({
+      error: null,
+      errorSteps: [],
```

Comment:
need to remove errorSteps and do it in render() so error on the breadcrump happen at the correct place (3 or 4 based on the error)